rest/vx.go

```diff
@@ -25,10 +25,7 @@ type VXClient struct {
 //
 // This method returns an iterator that should be used to access the results via this pattern:
 //
-//	iter, err := c.ListStockFinancials(context.TODO(), params, opts...)
-//	if err != nil {
-//		return err
-//	}
+//	iter := c.ListStockFinancials(context.TODO(), params, opts...)
 //	for iter.Next() {
 //	    log.Print(iter.Item()) // do something with the current value
```

Comment:
why is the spacing changed on some of them but not this one?

Reply:
They look the same in my editor, but some comments were done with tabs and some were done with spaces it seems. Running `go fmt` didn't fix it so I just made sure that they were all indented with a tab. Let me know if they still don't look right after my last change.

Reply:
looks good now, thanks!